instrumented/benchmark/build.gradle.kts

```diff
@@ -107,7 +107,7 @@ reviewBenchmark {
 
     // Logs Benchmarks
     addThreshold(
-        "benchmark_create_one_log", 500
+        "benchmark_create_one_log", 1
```

Comment:
What's the unit there ? milliseconds ?

Reply:
Yup...I resumed everything to milliseconds when I did the refactor ....there was too much noise because of all these transformations to nanos. I assumed that milliseconds will be enough and we can actually admit that 1 milli is the minimum threshold that we are interested in.